webpages/settings/index.js

```diff
@@ -362,6 +362,15 @@ const vue = (window.vue = new Vue({
       const matchesEasterEgg = addonManifest.tags.includes("easterEgg")
         ? this.selectedTab === "easterEgg" || addonManifest._enabled
         : true;
+
+      // April fools
+      if (addonManifest._addonId === "dango-rain") {
+        const now = new Date().getTime() / 1000;
+        if (this.selectedTab === "easterEgg") {
+        } else if (now < 1617364800 && now > 1617192000) {
```

Comment:
The `now < 1617364800 && ` will hide the addon behind the komoni code after April fools even if it is enabled. Remove it pls.

Reply:
this was intended by W_L.

Reply:
Could we show it if they have force after aprill fools?